Documentation/adding-new-repository.md

```diff
@@ -41,6 +41,30 @@ When applicable, prepare Docker image repositories for the new component.
   * robots: Read & Write
 * For closed source repository, create a new private repository in registry.mender.io
 
+## Makefile
+
+Consider adding a Makefile to the repository. This will serve both developers on their
+local environment and automated templated jobs on CI.
+
+A Makefile in a Mender repository is expected to contain at least the following targets:
+
+* `fmt`: Format the code. Typically using `gofmt` for golang repositories
+* `lint`: Lint the code. On new golang repositories, we are trying to use `golangci-lint` tool.
+* `build`: Build repository binary in `bin/` directory.
```

Comment:
Should mention that this is the default target.

Reply:
I thought about it but is not the case in [latest Makefiles](https://github.com/mendersoftware/deviceconnect/blob/master/Makefile#L21). @alfrunes do you oppose to changing the default target in the existing Makefiles to only build?

Reply:
This is a mistake on my part, `build` should definitely be the default target.

Reply:
Good, I've seen your PRs. I'll update the doc here accordingly